tests/test_mixin.py

```diff
@@ -46,6 +46,9 @@ def test_as_meta(self):
             'twitter_description': 'post meta',
             'gplus_type': 'Article',
             'title': 'a title',
+            'og_title': 'a title',
+            'twitter_title': 'a title',
```

Comment:
Can we use different title strings here? This will ensure we are checking the correct title